src/components/Auth/SignIn.tsx

```diff
@@ -79,43 +79,45 @@ const SignIn = () => {
             type='password'
             required
           />
-          <Flex justifyContent='center' align='center' mb='24px'>
-            <CustomCheckbox formValue='keepLogedIn' label={t('keep_me_logged', { defaultValue: 'Kepp me logged' })} />
+          <Flex justifyContent='center' align='center'>
+            <CustomCheckbox formValue='keepLogedIn' label={t('keep_me_logged', { defaultValue: 'Keep me logged' })} />
 
             <NavLink to='/auth/forgot-password'>
-              <Text color={textColorBrand} fontSize='sm' w='124px' fontWeight='500'>
+              <Text color={textColorBrand} fontSize='sm' fontWeight='500' whiteSpace='nowrap'>
                 {t('forgot_password')}
               </Text>
             </NavLink>
           </Flex>
-          <Button type='submit' fontSize='sm' variant='brand' fontWeight='500' w='100%' h='50' mb='24px'>
+          <Button type='submit' fontSize='sm' variant='brand' fontWeight='500' w='100%' h={50}>
             {t('signin')}
           </Button>
-        </Box>
+        </Flex>
       </FormProvider>
 
-      <Flex flexDirection='column' justifyContent='center' alignItems='start' maxW='100%' mt='0px'>
-        <Text fontWeight='400' fontSize='14px'>
+      <Flex flexDirection='column' justifyContent='center' alignItems='start' maxW='100%' mt={0}>
+        <Text fontWeight='400' fontSize='sm'>
           {t('not_registred_yet')}
           <NavLink to='/signup'>
-            <Text color={textColorBrand} as='span' ms='5px' fontWeight='500'>
+            <Text color={textColorBrand} as='span' ms={1} fontWeight='500'>
               {t('create_account')}
             </Text>
           </NavLink>
         </Text>
       </Flex>
-      <Box pt={2}>
-        <FormControl isInvalid={isError}>
-          {isError && <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>}
-        </FormControl>
-      </Box>
+      {isError && (
+        <Box>
+          <FormControl isInvalid={isError}>
+            <FormErrorMessage>{error?.message || 'Error al realizar la operación'}</FormErrorMessage>
```

Comment:
I think we should add the default error message into translations and use it everywhere